src/resources/postcss/components/full/_events-bar.pcss

```diff
@@ -9,14 +9,30 @@
 	.tribe-events-header--has-event-search .tribe-events-c-events-bar {
 
 		.tribe-common--breakpoint-medium& {
-			background-color: var(--color-background);
+			background-color: var(--tec-color-background-events-bar);
 		}
 	}
 
 	.tribe-events-c-events-bar--border {
 
 		.tribe-common--breakpoint-medium& {
-			border: 1px solid var(--color-border-secondary);
+			border: 1px solid var(--tec-color-border-events-bar);
+		}
+	}
+
+	.tribe-common-c-btn.tribe-events-c-search__button {
```

Comment:
just checking if this extra specificity is needed. also this should go in `src/resources/postcss/components/full/_search.pcss`